cocos/core/gfx/webgl/webgl-device.ts

```diff
@@ -789,4 +799,10 @@ export class WebGLDevice extends GFXDevice {
         gl.colorMask(true, true, true, true);
         gl.blendColor(0.0, 0.0, 0.0, 0.0);
     }
+
+    private _onWebGLContextLost (event: Event) {
+        warnID(11000);
+        warn(event);
+        event.preventDefault();
```

Comment:
@PPpro  Please verify if the `preventDefault` is implemented on all adatpers.

Reply:
'webglcontextlost' event not supported in minigame platforms